package.json

```diff
@@ -49,24 +49,26 @@
   "homepage": "https://github.com/apollostack/graphql-tools#readme",
   "dependencies": {
     "deprecated-decorator": "^0.1.6",
+    "lodash": "^4.17.4",
```

Comment:
Hmmm for the final release it would be super awesome to not include lodash - for client side use that really kills the bundle size.